newscoop/src/Newscoop/GimmeBundle/Controller/IssuesController.php

```diff
@@ -50,8 +49,8 @@ public function getIssuesAction(Request $request)
         }
 
         $paginator = $this->get('newscoop.paginator.paginator_service');
-        $issues = $paginator->paginate($issues, array(
-            'distinct' => false
+        $issues = $paginator->paginate($issues->getResult(), array(
```

Comment:
in IssueRepository we do left joins to get also language and sections and it get messed by serializer and will not display all issues properly, but only one. I could add a new param if we want to make left joins or not, because serializer seems to complain if we have query with joins.